src/github-handlers/private-notifications/notification-message.ts

```diff
@@ -0,0 +1,71 @@
+export class NotificationMessage {
+  private static _formatMessage(template: string, data: Record<string, string>): string {
+    // "Parameter name `_` must match one of the following formats: strictCamelCase"
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    return template.replace(/{{\s*(\w+)\s*}}/g, (_, key) => data[key] || "");
+  }
+
+  static getRfcMessage(data: { username: string; comment: string; commentUrl: string }): string {
+    const template = `<b>Hello {{ username }}</b>,
+    
+It seems you have not responded to the RFC comment yet. Please provide your feedback on the proposed changes:
+  
+{{ comment }}
```

Comment:
Much more concise to just wrap this in a clickable link instead of adding the whole sentence after.